resin/base_request.py

```diff
@@ -26,6 +26,10 @@ def __init__(self):
         self.token = Token()
         self.util = Util()
 
+    @property
+    def timeout(self):
+        return self.settings.get('timeout')
```

Comment:
Everything in settings is a `str` - do we need to convert this back to an `int` here?

Reply:
Good catch, fixed